index.html

```diff
@@ -1,224 +1,305 @@
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100&display=swap"
+      rel="stylesheet"
+    />
     <title>Portfolio</title>
-    <link rel="icon" type="image/x-icon" href="images/favicon.ico">
-    <link rel="stylesheet" href="styles.css">
-</head>
-<body>
+    <link rel="icon" type="image/x-icon" href="Images/favicon.ico" />
+    <link rel="stylesheet" href="css/styles.css" />
+    <link rel="stylesheet" href="css/media.css" />
+  </head>
+  <body>
     <!-- header section -->
     <header>
-        <nav class="headline">
-            <a href="#">
-                <h4>Kifle Haile</h4>
-            </a>
-            <a>
-                <div>
-                    <img src="Images/Menu.png" alt="icon">
-                    <img src="Images/Menu.png" alt="icon">
-                    <img src="Images/Menu.png" alt="icon">
-                </div>
-            </a>
-        </nav>
-        <!-- headline section -->
-        <section class="headline_section">
-            <div>
-                <h1 class="h1">Hey There. I'm Kifle</h1>
+      <nav class="headline">
+        <a>
+          <h4>Kifle Haile</h4>
+        </a>
+        <img src="Images/Menu.png" alt="icon" />
 
-                <h4 class="h4">I'm a Software Developer</h4>
-
-                <p class="p">
-                    I can help you build a product , feature or website. Look through some of my work and experience! If
-                    you like what you see and have a project you need coded,
-                    don’t hesitate to contact me. </p>
-                <!-- social media section -->
-                <ul class="image-icons">
-                    <li><a href="https://github.com/kifle23"><img src="Images/GitHub.png" alt="GitHub"></a></li>
-                    <li><a href="https://www.linkedin.com/in/kifle-haile-5a613761"><img src="Images/Linkedin.png"
-                                alt="Linkedin"></a></li>
-                    <li><a href="https://twitter.com/KifleHaile12"><img src="Images/Twitter.png" alt="Twitter"></a></li>
+        <ul class="nav">
+          <li><a href="#resent-works">Portfolio</a></li>
+          <li><a href="#about-section">About</a></li>
+          <li><a href="#contact-detail">Contact</a></li>
+          <li><a class="contact-icon" href="#contact-detail"></a></li>
+        </ul>
+      </nav>
+      <!-- headline section -->
+      <section id="headline_section" class="headline_section">
```

Comment:
- [ ] [OPTIONAL] I found out that the `headline section` is wrapped inside the `header` tag which is not a correct implementation of HTML semantics. I highly recommend removing the `headline section` from the `header` tag. 😄 
- [ ] This is a provided link for [HTML semantics](https://developer.mozilla.org/en-US/docs/Glossary/Semantics).